ReisTecnologia/abc
# Docker dynamodb local

How to test:
Make sure to have docker desktop installed
Run the command `docker compose up`
Check to see if the container is running
